plugins/isp/internode_usage

```diff
@@ -104,6 +104,25 @@ fetch() {
 		    && false)
 }
 
+get_cached_api_xml() {
+	API_XML=''
```

Comment:
Probably this variable (especially with such a generic name) deserves to be `local`?